airflow/opt/providers/etna/etna/etls/metis_linker.py

```diff
@@ -129,12 +132,16 @@ def data_frame_update(self, model_name, script, tail, update, metis, model):
             else:
                 data = data.set_index(id, drop=True)
                 for name, attributes in data.T.to_dict().items():
-                    if not self.get_identifier(model_name, name):
+                    if model_name in self.rules and not self.get_identifier(model_name, str(name)):
                         continue
                     update.update_record(model_name, name, {k: v for k,v in attributes.items() if v is not None})
 
     def file_collection_update(self, model_name, script, tail, update):
+        if model_name not in self.rules:
+            raise f"Cannot filter by file without a rule for {model_name}"
+
```

Comment:
however, we can't skip for files without a rule and must raise.

Reply:
Honestly, might as well split out a "rule for this model?" check early and just raise f"Cannot link data for a model without a rule {model_name}".  Or do we long-term plan to continue supporting identifier validation external to gnomon?

Reply:
nvm!  We do want to support identifier validation external to gnomon